src/services/shell.service.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+/**
+ * Thin wrapper around electron `shell`, and other shell-like functions
+ */
+import { shell } from 'electron';
+import launchEditor from 'react-dev-utils/launchEditor';
+
+import type { Project } from '../types';
+
+export const openProjectInFolder = (project: Project) =>
+  shell.openItem(project.path);
+
+export const openProjectInEditor = (project: Project) =>
+  launchEditor(project.path, 1, 1);
```

Comment:
While these wrappers feel silly, we'll need them in multiple places (the ProjectPage work from @AWolf81), and I like the idea of having 1 code path for both places

Reply:
I think it would be also OK to inline it as it's just executing one command from a library.
But having one location is better if we need to change something later as we need to edit only one location and we don't have to know that it's used twice.

So I don't have a strong opinion - both is OK.

Reply:
Should shell.moveItemToTrash (the delete project functionality from `delete-project.saga.js`) go in here too, for consistency?

Reply:
> Should shell.moveItemToTrash (the delete project functionality from delete-project.saga.js) go in here too, for consistency?

Good idea!